Library/Homebrew/cmd/audit.rb

```diff
@@ -375,11 +375,11 @@ def audit_homepage
   end
 
   def audit_specs
-    if head_only?(formula) && formula.tap.downcase != "homebrew/homebrew-head-only"
+    if head_only?(formula) && formula.tap.downcase != "*/homebrew-head-only"
```

Comment:
Sorry - how can I make this work? How can I compare in ruby with wildcard pattern?